firebase-web/src/main/java/io/spine/web/firebase/subscription/UpdatePayload.java

```diff
@@ -136,7 +140,30 @@ private static String key(Message id) {
      */
     NodeValue asNodeValue() {
         NodeValue node = NodeValue.empty();
-        messages.forEach((id, message) -> node.addChild(id, StoredJson.encodeOrNull(message)));
+        messages.forEach((id, message) -> addChildOrNull(node, id, message));
         return node;
     }
+
+    /**
+     * Adds an {@code <ID:message>} key-value pair as a child of the given node.
+     *
+     * <p>If the message is {@link Empty}, adds a {@code null} child under the given key,
+     * effectively deleting it from the database.
+     */
+    private static void addChildOrNull(NodeValue node, String id, Message message) {
+        boolean isEmpty = Empty.getDefaultInstance()
+                               .equals(message);
+        if (isEmpty) {
+            node.addNullChild(id);
+        } else {
+            node.addChild(id, StoredJson.encode(message));
+        }
+    }
+
+    /**
+     * An identity {@link Event}-to-{@link Message} conversion.
+     */
+    private static Function<Event, Message> identity() {
```

Comment:
Please use `java.util.Function.identity()` instead.

Reply:
@dmitrykuzmin, please revisit this comment.

Reply:
As discussed vocally, I've added a clarifying comment in the code about why we don't use the standard `Function.identity()`.